### DISCUSSION TITLE
Realtime connection reset

### DISCUSSION THREAD

**Discussion by bogab** [supabase/supabase#4014] [Questions] [answered]
Why does the realtime connection reset every minute?
I am missing some realtime updates, could this be the reason?
![Screenshot from 2021-11-23 13-47-42](https://user-images.githubusercontent.com/6377968/143020618-3535f66d-4e83-4bad-9ed6-34756dd58bc7.png)

**Comment by w3b6x9** [comment, marked as answer]
@bogab it appears you're using `supabase-js` client. There was a bug that was causing this. Can you please upgrade to the latest version?

**Reply by w3b6x9** [reply]
@bogab please reach out if the above doesn't fix your connection reset issue
